src/darker/black_diff.py

```diff
@@ -82,14 +82,25 @@
 logger = logging.getLogger(__name__)
 
 
-def run_black(src: Path) -> Tuple[List[str], List[str]]:
+def run_black(src: Path, black_args: dict) -> Tuple[List[str], List[str]]:
     """Run the black formatter for the contents of the given Python file
 
     Return lines of the original file as well as the formatted content.
 
+    :param black_args: Command-line arguments to send to ``black.FileMode``
```

Comment:
I've tried to follow PEP8 docstring conventions by having an empty line before the closing triple-double-quote:
```suggestion
    :param black_args: Command-line arguments to send to ``black.FileMode``

```